ajax/libs/imgcache.js/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "imgcache.js",
+  "version": "2.0.0",
+  "description": "JS library based on the File API to cache images for offline recovery (target: cordova/phonegap & chrome)",
+  "filename": "imgcache.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/chrisben/imgcache.js.git"
+  },
+  "keywords": [
+    "image",
+    "cache",
+    "html5",
+    "file",
+    "filesystem",
+    "offline"
```

Comment:
Let's add more keywords, such as https://github.com/chrisben/imgcache.js/blob/master/bower.json#L17-L18 & https://github.com/chrisben/imgcache.js/blob/master/bower.json#L20
This will useful to people search the lib via our CDNJS website engine.
Thanks